integration-tests/src/test/resources/python-scripts/wlst-create-domain-onpv.py

```diff
@@ -50,8 +50,8 @@ def create_domain():
   st=create(template_name, 'ServerTemplate')
   print('Done creating server template: %s' % template_name)
   cd('/ServerTemplates/%s' % template_name)
-  print('Set managed server port in template')
-  cmo.setListenPort(int(managed_server_port))
+  print('(SKIP) Set managed server port in template')
+  #cmo.setListenPort(int(managed_server_port))
```

Comment:
remove the commented out lines.

Reply:
Added one more line of comment to describe the reason
----
print('(SKIP) Set managed server port in template')
#If no explicit listenport is set, operator infra will pick 7100
#cmo.setListenPort(int(managed_server_port))
----